.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ orbs:
     executors:
       operator-build:
         docker:
-          - image: orangeopensource/casskop-build:v0.9.0
+          - image: orangeopensource/casskop-build:v0.15.0-pr137
```

Comment:
Temporary version ?

Reply:
Yes, until this issue will be fixed : https://github.com/operator-framework/operator-sdk/issues/2485